diff -Naurp pl-5.10.5/src/os/pl-stream.c pl-5.10.5.oden/src/os/pl-stream.c
--- pl-5.10.5/src/os/pl-stream.c	2011-08-19 14:31:28.000000000 +0000
+++ pl-5.10.5.oden/src/os/pl-stream.c	2011-11-04 13:50:12.000000000 +0000
@@ -111,7 +111,7 @@ typedef wchar_t pl_wchar_t;
 
 #define char_to_int(c)	(0xff & (int)(c))
 
-#define TMPBUFSIZE 256			/* Serror bufsize for Svfprintf() */
+#define TMPBUFSIZE 4096			/* Serror bufsize for Svfprintf() */
 
 int Slinesize = SIO_LINESIZE;		/* Sgets() buffer size */
 
diff -Naurp pl-5.10.5/src/swipl-ld.c pl-5.10.5.oden/src/swipl-ld.c
--- pl-5.10.5/src/swipl-ld.c	2011-08-19 14:31:28.000000000 +0000
+++ pl-5.10.5.oden/src/swipl-ld.c	2011-11-04 13:50:12.000000000 +0000
@@ -949,18 +949,18 @@ fillDefaultOptions()
 static void
 getPrologOptions()
 { FILE *fd;
-  char cmd[512];
+  char cmd[4096];
 
   sprintf(cmd, "%s --dump-runtime-variables", pl);
   if ( verbose )
     printf("\teval `%s`\n", cmd);
 
   if ( (fd = popen(cmd, "r")) )
-  { char buf[256];
+  { char buf[4096];
 
     while( fgets(buf, sizeof(buf), fd) )
-    { char name[100];
-      char value[256];
+    { char name[4096];
+      char value[4096];
       char *v;
 
       if ( sscanf(buf, "%[^=]=%[^;\n]", name, value) == 2 )
